### ISSUE TITLE
Can't compatible with torch list setting.

### ISSUE THREAD

**Issue by Greygreygod** [Nyphet/torch-hit#10]
I want to make function only work with "Consecration"mod's firestick,so i just leave it on torch list setting,like ""torch list" = ["consecration:fire_stick"]".
But i found torch still work with the function,it's ok that both work,but it will make player using torch more than firestick.It's not what I wanted it to be.

**Comment by Crystal-Spider** [comment]
As of now that configuration option is only to allow other items other than torches to work, which means that it's a list of extra items that will trigger the "fire attack". Regardless of that list, Vanilla torches always work to set on fire.

I could change how that works, or even add another configuration option to allow control on whether Vanilla torches should be included or not. However this I'm afraid may be out of scope with the aim of the mod, which should be just to make a fire attack with torches.

It's also true that I'm not against giving more freedom of choice to the end user of the mod, so I'll think about adding this feature and update you here.

**Comment by Greygreygod** [comment]
Alright,I just found it's a great mod to combine with Consecration mod,because it's a good way to burn undead in early game(firestick from Consecration mod are too frail to use XP).Torch to burn undead also can be a choice,maybe i can just change the recipes about torch.
Follow your thoughts,it's a nice mod XD

**Comment by Crystal-Spider** [comment]
Don't worry about closing the issue, I don't dislike the suggestion and I need the issue open to remind me about adding this feature with the next version.  
If I eventually decide on not adding the feature, I will update you here and close the issue myself.

In the meantime I thank you for the feature suggestion, the effort in searching for alternative solutions and the kind words.

**Comment by Greygreygod** [comment]
OK,thank you :D

**Comment by Crystal-Spider** [comment]
I've decided on adding a new configuration option, `vanilla torches enabled`. This new configuration option allows to enable (default) or disable Vanilla torches.  
When updating, mind the fact I also changed the name of some other already existing configuration options.  
If you want a complete list of changes, refer to the changelog.

Version 6.0.0.0 will soon be available for all active Minecraft versions (1.16.5, 1.18.2, 1.19.2, 1.19.3).
